vgaidarji/github-actions-pr-merger
# Print to console github payload

### What has been done
1. Event `issue_comment` is not triggering checks on PR level since it's tied to any git ref. Instead workflow is triggered at [repository Actions](https://github.com/vgaidarji/github-actions-pr-merger/actions) level.
See https://github.community/t5/GitHub-Actions/on-issue-comment-events-are-not-triggering-workflows/td-p/37177

### How to test
1. Merge PR to `master` since `issue_comment` runs a workflow only when changes are merged to `master` branch
2. Open next PR or comment on any PR and make sure workflow is triggered in [repository Actions](https://github.com/vgaidarji/github-actions-pr-merger/actions)
